docs/proposals/kubeedgeadm-scope.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Kubeedgeadm kubeEdge installer scope (Issue 324)
+status: Pending
+authors:
+    - "@samy2019"
+approvers:
+  - "@m1093782566"
+  - "@rohitsardesai83"
+  - "@sids-b"
+creation-date: 2019-04-11
+last-updated: 2019-04-12
+---
+
+## Motivation
+Many users shared their feedback that kubeEdge installation is too complicated and it may prevent people from trying kubeEdge.
+There should a simplified way to have **Getting Started with KubeEdge**, so that user can concentrate more on using it instantly, rather than
+getting entangled in the installation steps.
+
+## Proposal
+KubeEdge shall have simple commands and steps to bring up both cloud and edge components.
+The user experience in **Getting Started with KubeEdge** will be seamless.
+Hence proposing the following commands for KubeEdge installation process.
```

Comment:
Can we have something like this for kubeedge installer ?

```sh
Usage:
 kubeedge [module] [command]

Available Modules:
cloud
edge

Available Commands:
  help      Help about any command
  init        Run this command in order to set up the kubeedge cloud.
  join       Run this on any edge node you wish to join an existing cluster.
  reset      Run this to revert any changes made to this host by 'kubeedge init' or 'kubeedge join'.
  version   Print the version of kubeedge
```

Reply:
Yes, It is the plan which i already have. 
Once it is done, shall show a demo to you all.

Reply:
lets have it in the design as well.

Reply:
Good point, @rohitsardesai50 

I think these commands are enough for 1st stage.

Reply:
Have added the basic, required one's. Please check the updated document.